pipelines/imputation/testing/LiftoverVcfs.wdl

```diff
@@ -0,0 +1,99 @@
+version 1.0
+
+# Liftover VCFs from hg19 to hg38
+workflow LiftoverVcfs {
```

Comment:
should this be in the scientific validation folder instead? I dont think it belongs in the testing folder at least.

Reply:
yeah it sort of doesn't belong anywhere, but happy to move to scientific validation - agree that makes more sense than testing